content/go/concepts/slices/slices.md

```diff
@@ -0,0 +1,125 @@
+---
+Title: 'Slices'
+Description: 'Slices in Go are similar to arrays, but with the flexibility of being able to change their size.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Slices'
+  - 'Arrays'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+**Slices** in Go point to a section of an array. They are similar to arrays, but with the flexibility of being able to change their size. Care needs to be taken when modifying elements in slices that point to an existing array, as the element value will change in the underlying array. Slices have a length and capacity.
+
+The length is the number of elements in the slice and can be found using `len(slice_name)`.
+
+The capacity can be found using `cap(slice_name)`. It is the number of elements in the underlying array from the slice's start index to the end of the array.
+
+## Creating and Initializing Slices
+
+### Declare and Initialize a New Slice
+
+#### Syntax
+
+```pseudo
+new_slice := []<datatype>{<values>}
+```
+
+#### Example
+
+A slice containing integers, the length and capacity will both be 5.
+
+```go
+  s := []int{1,2,3,4,5}
+```
+
+### A Slice from an Existing Array
+
+A slice can be created from another array or slice.
+
+#### Syntax
+
+```pseudo
+new_slice_from_array := <array_name>[<from_index>:<to_index>]
+```
```

Comment:
Again, a sentence describing the syntax given would be good.

Reply:
@SSwiniarski I added in this sentence:

> A slice can be created from another array (or slice) called `<array_name>`. The slice will contain elements from `<array_name>[from_index]` up to (but not including) `<array_name>[to_index]`. `<from_index>` and `<to_index>` are optional. If `<from_index>` is left blank, then the slice will start from the first element of `<array_name>`. If `<to_index>` is left blank, the slice will end at the last element of `<array_name>`.